src/modules/PackageSlideShow.rb

```diff
@@ -540,6 +540,9 @@ def SetCurrentCdNo(src_no, cd_no)
       SlideShow.CheckForSlides
       FindNextMedia()
 
+      # do not rebuild if the user reads the release notes
```

Comment:
Hi, in general I do not like whole this piece of code ( nothing against your change ). It is from my POV touching internals of SlideShow module. I think ideally whole this part that basically only call various parts of SlideShow should live in Slideshow itself. ( including above called CheckForSlides ). Method like Redraw would be good.